skills/timezone.js

```diff
@@ -1,63 +1,94 @@
-/*
+const {findTimeZone, getZonedTime} = require('timezone-support');
+const {formatZonedTime} = require('timezone-support/dist/parse-format');
+const ExpressionList = require('../submodules/Regex/ExpressionList');
+const timeZoneExList = require('../submodules/Regex/timeZoneExList');
 
- This simply hears the time in United States timezones and reports the system time (EST) and adjusts it accordingly
+// Zone Name comes from the npm module timezone-support and pulled from a list listTimeZones.
+// Using zone names will account for Daylights Savings Time
+const zones = {
+  PST: 'America/Los_Angeles',
+  MST: 'America/Phoenix',
+  CST: 'America/Chicago',
+  EST: 'America/New_York',
+  IST: 'Indian/Kerguelen',
+  GMT: 'Europe/London',
+  CEST: 'Europe/Warsaw',
+  CTC: 'Asia/Shanghai'
+};
 
-*/
+const adjustForTimeZone = (timeZone) => {
+  const zone = findTimeZone(timeZone);
+  const localTime = new Date();
+  const zoneTime = getZonedTime(localTime, zone);
+  const format = 'h:mm A z'
```

Comment:
missing semicolon, can you run eslint on this file? It should add it for you with `--fix`